### PR TITLE
use gh cli instead of create pr action for initialize

### PR DESCRIPTION
**Pull request by kommendorkapten**
#### Summary
Use gh cli instead of an action to create pull requests

#### Release Note
N/A

#### Documentation
N/A

### CODE REVIEW

File: .github/workflows/initialize.yml [hunk 1]
```diff
@@ -147,20 +147,15 @@ jobs:
...
-          branch: init-root-targets
-          base: ${{ inputs.branch }}
-          signoff: true
-          draft: ${{ inputs.draft }}
-          reviewers: asraa,dlorenc,haydentherapper,joshuagl
-          token: ${{ secrets.SIGSTORE_ROOT_SIGNING_FINE_GRAINED_PAT }}
+        env:
+          DRAFT: ${{ inputs.draft && '-d' || '' }}
+        run:
+          GH_TOKEN=${{ secrets.SIGSTORE_ROOT_SIGNING_FINE_GRAINED_PAT }} gh pr create -B ${{ inputs.branch }} -H init-root-targets -t "Update Root and Targets to version ${{ env.VERSION }}" -b "Initializes a new root and targets to version ${{ env.VERSION }}" ${DRAFT} -r kommendorkapten -r dlorenc -r haydentherapper -r joshuagl
```
Comment (kommendorkapten) [inline .github/workflows/initialize.yml]: Splitting this to multiple lines with `\` didn't work 😞

File: .github/workflows/reuseable-snapshot-timestamp.yml [hunk 2]
```diff
@@ -191,28 +191,17 @@ jobs:
...
-          commit-message: update timestamp
-          title: Update Timestamp
-          body: Sign timestamp file
-          branch: update-snapshot-timestamp
-          signoff: true
-          reviewers: bobcallaway,haydentherapper,kommendorkapten
-          token: ${{ secrets.token || secrets.GITHUB_TOKEN }}
-          base: ${{ inputs.branch }}
+        run:
+          git commit -s --allow-empty -m "Upddate timestamp"
```
Comment (bobcallaway) [inline .github/workflows/reuseable-snapshot-timestamp.yml]:
```suggestion
          git commit -s --allow-empty -m "Update timestamp"
```

File: .github/workflows/reuseable-snapshot-timestamp.yml [hunk 3]
```diff
@@ -191,28 +191,17 @@ jobs:
...
-          commit-message: update snapshot and timestamp
-          title: Update Snapshot and Timestamp
-          body: Sign snapshot and timestamp files
-          branch: update-snapshot-timestamp
-          signoff: true
-          reviewers: bobcallaway,haydentherapper,kommendorkapten
-          token: ${{ secrets.token || secrets.GITHUB_TOKEN }}
-          base: ${{ inputs.branch }}
+        run:
+          git commit -s --allow-empty -m "Upddate snapshot and timestamp"
```
Comment (bobcallaway) [inline .github/workflows/reuseable-snapshot-timestamp.yml]:
```suggestion
          git commit -s --allow-empty -m "Update snapshot and timestamp"
```

**Review by bobcallaway — DISMISSED** [review]
lgtm, fyi @haydentherapper

File: .github/workflows/initialize.yml (hunk 1 above)
Comment (Hayden-IO) [inline .github/workflows/initialize.yml]: Can you update this list to be just Bob, you, and me? This is aligned with other notifications

**Review by Hayden-IO — COMMENTED** [review]
What’s the motivation for the change, given 
 we want to switch to TUF on ci?

**Comment by Hayden-IO** [comment]
Oh I just saw the other thread on the action upgrade, LGTM on the context

**Review by bobcallaway — APPROVED** [review]

File: .github/workflows/initialize.yml [hunk 4]
```diff
@@ -147,20 +147,15 @@ jobs:
           git config --global user.email "noreply@github.com"
           git config --global user.name "GitHub"
 
+          git checkout -b init-root-targets
           # Commit the REPO changes
           git add ${REPO}
           git commit -s -m "Add staged repository metadata"
+          git push origin HEAD
```
Comment (Hayden-IO) [inline .github/workflows/initialize.yml:154]: @kommendorkapten Is HEAD correct here? Or is this supposed to be a branch?

Comment (kommendorkapten) [reply .github/workflows/initialize.yml:154]: @haydentherapper `HEAD` is correct here, as it refers to the active branch `init-root-targets`, which is the one we will create the PR from. Does this cause any problems?

Comment (Hayden-IO) [reply .github/workflows/initialize.yml:154]: No issue, just wanted to double check.

**Pull request merged**